### ISSUE TITLE
WxWgpuWindow object has no attribute 'add_event_handler'

### ISSUE THREAD

**Issue by cansik** [pygfx/wgpu-py#485]
At least on MacOS, it seems that the `WxWgpuWindow` integration seems not to work. The [example found](https://pygfx.readthedocs.io/en/stable/_gallery/other/cube_wx.html) here can not be run because of the following issue:

```python
Traceback (most recent call last):
  File "/pyfx_image_viewer.py", line 11, in <module>
    canvas = WgpuCanvas()
  File "/venv/lib/python3.9/site-packages/wgpu/gui/wx.py", line 146, in __init__
    self._subwidget.add_event_handler(weakbind(self.handle_event), "*")
AttributeError: 'WxWgpuWindow' object has no attribute 'add_event_handler'
```

Versions:
- pygfx `0.1.18`
- wgpu `0.15.0`
- wxpython `4.2.1`
- python `3.9.18`

**Comment by cansik** [comment]
@Korijn I have created a PR to fix this in wgpu and another one which updates the wx example in pygfx. Would be great if you could have a look.

**Comment by cansik** [comment]
@Korijn Would it be possible to create a new wgpu release, since in `0.15.1` my changes of the merge request #486  are not present? This blocks using wgpu & pygfx together with wx.

**Comment by almarklein** [comment]
#496

**Comment by almarklein** [comment]
new release is available. Thanks for the heads-up!

**Comment by almarklein** [comment]
Closed by #552
